docs/sources/getting-started/getting-started-sql.md

```diff
@@ -0,0 +1,90 @@
++++
+title = "With Grafana and MS SQL"
+description = "Guide for getting started with Grafana"
+keywords = ["grafana", "intro", "guide", "started"]
+aliases = ["/docs/grafana/latest/guides/gettingstarted","/docs/grafana/latest/guides/getting_started"]
+weight = 400
++++
+
+# Getting started with Grafana and MS SQL Server
+
+Microsoft SQL Server is a popular relational database management system that is widely used in development and production environments. Grafana v5.1+ ships with a built-in Microsoft SQL Server (MSSQL) data source plugin that allows you to query and visualize data from any Microsoft SQL Server 2005 or newer, including Microsoft Azure SQL Database. 
+
+This topic walks you through the steps to create a series of dashboards in Grafana to display metrics from a MS SQL Server database. You can also configure the MS SQL Server data source on a [Grafana Cloud](https://grafana.com/docs/grafana-cloud/) instance without having to host Grafana yourself.
+
+## Step 1: Install Grafana and build your first dashboard
+
+Use the instructions in [Getting started with Grafana]({{< relref "getting-started.md" >}}) to:
+- Install Grafana.
+- Log in to Grafana.
+- Create your first dashboard.
+
+## Step 2: Download MS SQL Server
+
+MS SQL Server can be installed on many different operating systems. Refer to the [MS SQL Server downloads page](https://www.microsoft.com/en-us/sql-server/sql-server-downloads), for a complete list of all available options.
+
+Alternately, install MS SQL Server using the resources available in [grafana/grafana](https://github.com/grafana/grafana) GitHub repository (recommended). Here you will find a collection of supported data sources, including MS SQL Server, along with test data and pre-configured dashboards for use.
+
+> **Note:** Installing MS SQL Server on Windows from the [grafana/grafana](https://github.com/grafana/grafana/tree/master/devenv) GitHub repository is not supported at this time.
+
+## Step 3: Install MS SQL Server
+
+You can install MS SQL Server on the host running Grafana or on a remote server. To install the software from the [downloads page](https://www.microsoft.com/en-us/sql-server/sql-server-downloads), follow their setup prompts.
+
+Otherwise, follow the instructions below to install and configure MS SQL Server from the [grafana/grafana](https://github.com/grafana/grafana/tree/master) repository.
+
+1. Clone the [grafana/grafana](https://github.com/grafana/grafana/tree/master) repository to your local system.
+
```

Comment:
Why are there empty lines between each list step?

Reply:
Removed empty lines.

Reply:
Weird. It's not showing that on my screen.